.devcontainer/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:24.04
+
+RUN DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get update && \
+    apt-get install -y \
+    adwaita-qt \
+    alsa-utils \
+    build-essential \
+    clangd \
+    gdb \
+    git \
+    libadwaitaqt1 \
+    pipewire \
+    pipewire-alsa \
+    pulseaudio-utils \
+    python3-pip \
+    qt6-wayland \
+    sudo \
+    udev \
+    vim && \
```

Comment:
Are all these packages developer tools, or do some belong to debian_buildenv.sh instead?

Reply:
Good question - most of these packages are expected to already be installed on the machine (git, udev for example), some others provide an opinionated out of the box experience (vim, clangd, gdb).

I think it makes sense to keep it separated but open to move some of these packages in the script.